FrontEnd/FrontEnd.Web/Components/Pages/Recipes/Form.razor

```diff
@@ -0,0 +1,204 @@
+﻿@using System.ComponentModel.DataAnnotations
+@rendermode InteractiveServer
+
+<EditForm Model="@FormData" OnValidSubmit="@HandleValidSubmit" FormName="recipe" novalidate>
+    <ObjectGraphDataAnnotationsValidator/>
+
+    <FluentGrid AdaptiveRendering>
+        <FluentGridItem xs="12" sm="12" md="6">
+            <FluentStack Orientation="Orientation.Vertical">
+                <div style="width: 100%;">
+                    <FluentTextField Name="title" @bind-Value="FormData.Title" Label="Title" Required style="width: 100%;"/>
+                    <FluentValidationMessage For="@(() => FormData.Title)"/>
+                </div>
+                <FluentTextArea Name="description"
+                                Rows="5"
+                                @bind-Value="FormData.Description"
+                                Label="Description"
+                                style="width: 100%;"/>
+                <FluentAutocomplete TOption="string"
+                                    AutoComplete="off"
+                                    Label="Tags"
+                                    Placeholder="Select tags"
+                                    OnOptionsSearch="OnTagsSearchAsync"
+                                    OptionText="@(tag => tag)"
+                                    @bind-SelectedOptions="@FormData.Tags"
+                                    Width="100%"/>
+                <div style="width: 100%;">
+                    <FluentNumberField Name="cookingTime"
+                                       @bind-Value="FormData.CookingTimeInMinutes"
+                                       Label="Cooking time (min.)"
+                                       Required
+                                       style="width: 100%;"/>
+                    <FluentValidationMessage For="@(() => FormData.CookingTimeInMinutes)"/>
```

Comment:
<img width="630" alt="Screenshot 2024-05-13 at 00 14 18" src="https://github.com/BenasB/vu-psk/assets/86420873/dd3c9d8c-c220-4fbf-b32c-cce6ff6797d7">

The field name is a variable name rather than label.

Reply:
Fixed up the parsing error, but it seems like there's no easy way to stop the nullability/required check from happening
![image](https://github.com/BenasB/vu-psk/assets/29711974/73f5cf66-43f8-4993-8ccb-3e7443835e26)